eventtracking/backends/caliper.py

```diff
@@ -1,12 +1,12 @@
-import logging
+"""
+Caliper event processing backend
+"""
 from celery.utils.log import get_task_logger
 
-from eventtracking.backends.routing import RoutingBackend
-
 LOGGER = get_task_logger(__name__)
 
 
-class CaliperBackend(RoutingBackend):
+class CaliperBackend:
     """
     Backend to transform events into xAPI compliant format
     and then route those events to configured endpoints.
```

Comment:
Need to add an `__init__` method here too. Otherwise throws error.

Reply:
I don't think we need `__init__` for either `XAPIBackend` or `CaliperBackend` at the moment. I add `__init__` to `XAPIBackend` while testing something. Let me remove it.

Reply:
We actually need these __init__ methods if we are going to pass the `OPTIONS` . Eventtracking pass these options to our backends while initializing those. And thus we will be needing these constructors.

Reply:
yeah we'll add those once we have `OPTIONS ` in-place. This PR assumes we don't have `OPTIONS `